libs/core/langchain_core/tracers/core.py

```diff
@@ -516,15 +516,19 @@ def __copy__(self) -> _TracerCore:
 
     def _end_trace(self, run: Run) -> Union[None, Coroutine[Any, Any, None]]:
         """End a trace for a run."""
+        return None
```

Comment:
due to https://github.com/PyCQA/flake8-bugbear/issues/301
Do we prefer to ignore the rule with `noqa` ?